src/flake8_to_ruff/isort.rs

```diff
@@ -0,0 +1,31 @@
+//! Extract isort configuration settings from a pyproject.toml.
+
+use std::path::Path;
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+// use crate::settings::types::PythonVersion;
+
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Default)]
+pub struct ISort {
```

Comment:
I think it would make sense to add a doc comment (a comment starting with `///`) linking the isort config documentation, so e.g.

```
/// The [isort configuration](https://pycqa.github.io/isort/docs/configuration/config_files.html).
```

Reply:
It's a nit but I think this can be `Isort` (rather than `ISort`).